posthog/models/entity.py

```diff
@@ -69,6 +69,16 @@ def equals(self, other) -> bool:
 
         return True
 
+    def includes(self, other) -> bool:
+        """ Checks if this entity is a generalized version of other. The ids match and the properties of (this) is a subset of the properties of (other)"""
```

Comment:
ok, starting to get confusing.

We want to go `1 PRECEDING` when the current entity is a superset of previous, right?

That implies checking `entities[i].is_superset(entities[i-1])`.

This does the reverse right now, doesn't it? 👀 

(Also, proposal on changing `includes` to `is_superset`/ `is_subset` ?

Reply:
for it to be a superset, it has less properties than (other) so it has a `subset` of (other)'s properties.

`self._filter.entities[i].includes(self._filter.entities[i - 1])` is being called in base.py which should be correct

Reply:
Yep, looks good now! Thanks for this change! :)

`includes` == `is_superset` was what seemed confusing to me. ( I get now, in retrospect, includes implied properties, not the entity)